.travis.yml

```diff
@@ -5,3 +5,9 @@ branches:
     - master
     - /^greenkeeper\/.*$/
 cache: yarn
+
+# Since the upgrade to Ubuntu Precise, yarn is not available to run scripts: `yarn run foo`
+# So let's install it manually
+before_install:
```

Comment:
I am confused, this link says that by default Yarn is present when yarn.lock is here.